modules/setting-higher-pid-limit-on-existing-cluster.adoc

```diff
@@ -15,6 +15,9 @@ Changing the `podPidsLimit` on an existing cluster will trigger non-control plan
 
 .Prerequisites
 
+* You have a ROSA Classic cluster.
++
+Configuring the `KubeletConfig` object is not supported on ROSA with hosted control planes (HCP).
```

Comment:
I see there are some related ROSA / HCP attributes in https://github.com/openshift/openshift-docs/blob/main/_attributes/attributes-openshift-dedicated.adoc#L45-L47. Can those be used here?